algorithms/yiddish.sbl

```diff
@@ -159,9 +162,7 @@ define mark_regions as (
 
 backwardmode (
   define R1 as $p1 <= cursor
-  // Like R1, but also allows the cursor to be outside R1 by the width of Tes.
-  define R1plus1 as $p1 <= cursor + sizeof '{Tes}'
-  // Like R1plus1, but by the width of Giml Yud Samekh
+  // Like R1, but by the width of Giml Yud Samekh
```

Comment:
I think this needs to incorporate the first part of the comment about R1plus, or else the meaning isn't clear without looking at the history.  I'll push a tweak for that.